src/jest/users.test.ts

```diff
@@ -76,3 +68,91 @@ describe('FIREBASE User tests', () => {
     await expect(getUserById(testUser1.id!)).rejects.toThrow(BaseError);
   });
 });
+
+/******************/
+/**** Express  ****/
+/******************/
+
+describe('EXPRESS user routes', () => {
+  let expressId: string;
+  let expressToken: string;
+  let hashedPassword: string;
+  const orgid = 1232344432 as number;
+  const user1: { [key: string]: any } = {
+    firstName: 'Thor',
+    lastName: 'Hansen',
+    email: 'express@mail.com',
+    password: 'Test123!!!',
+  };
+  test('Create user', async () => {
+    const res = await request(app)
+      .post('/users/' + orgid)
+      .send(user1);
+
+    expressId = res.body.id;
+    expressToken = res.body.token;
+    hashedPassword = res.body.password;
+    expect(res.statusCode).toBe(201);
+    expect(res.body.firstName).toBe(user1.firstName);
+    expect(res.body.lastName).toBe(user1.lastName);
+    expect(res.body.email).toBe(user1.email);
+    expect(res.body.password).toBeDefined();
+  });
+  test('Login user', async () => {
+    const res = await request(app)
+      .post('/users/' + orgid + '/login')
+      .send({
+        email: user1.email,
+        password: user1.password,
+      })
+      .set('Authorization', 'Bearer ' + expressToken);
+    expect(res.statusCode).toBe(200);
+    expect(res.body.user.firstName).toBe(user1.firstName);
+    expect(res.body.user.lastName).toBe(user1.lastName);
+    expect(res.body.user.email).toBe(user1.email);
+    expect(res.body.user.password).not.toBe(hashedPassword);
+  });
+  test('Get user by orgId', async () => {
+    const res = await request(app)
+      .get('/users/' + orgid)
+      .set('Authorization', 'Bearer ' + expressToken);
+    const user = res.body.find((e: User) => e.id === expressId);
+    expect(res.statusCode).toBe(200);
+    expect(user.firstName).toBe(user1.firstName);
+    expect(user.lastName).toBe(user1.lastName);
+    expect(user.email).toBe(user1.email);
+  });
+  test('Get user by token', async () => {
+    const res = await request(app)
+      .get('/users/' + orgid + '/me')
+      .set('Authorization', 'Bearer ' + expressToken);
+    expect(res.statusCode).toBe(200);
+    expect(res.body.firstName).toBe(user1.firstName);
+    expect(res.body.lastName).toBe(user1.lastName);
+    expect(res.body.email).toBe(user1.email);
+  });
+  test('Update user', async () => {
+    const user2: { [key: string]: any } = {
+      firstName: 'Thor',
+      lastName: 'Hansen',
+      email: 'updatedtest@mail.com',
+      password: 'Testå123Å4!!!!!',
+    };
+    const res = await request(app)
+      .put('/users/' + orgid + '/' + expressId)
+      .set('Authorization', 'Bearer ' + expressToken)
+      .send(user2);
+    expect(res.statusCode).toBe(200);
+    expect(res.body.firstName).toBe(user2.firstName);
+    expect(res.body.lastName).toBe(user2.lastName);
+    expect(res.body.email).toBe(user2.email);
+    expect(res.body.password).not.toBe(hashedPassword);
+  });
```

Comment:
Maybe also make a login after this request, to see if the password has been changed.

Reply:
done thanks for the idea <3